spec/factories/users.rb

```diff
@@ -4,5 +4,8 @@
     password { "password" }
     password_confirmation { "password" }
     organization
+
+    trait(:as_admin)  { role { :admin } }
+    trait(:as_user)   { role { :user } }
```

Comment:
I think id prefer `:user` just be the default, and only define a trait for admin. And Im not sure how I feel about the `as` prefix. `as_admin` reads to me like a macro or mixin we would use for a feature test, but for the factory I think `create(:user, :admin)` reads cleaner, IMO.

Reply:
How about using `:user` as a default role but keeping the trait for consistency?